TrafficSimulator/Assets/Car/AutoDrive.cs

```diff
@@ -374,6 +427,11 @@ public LaneNode CurrentNode
         {
             get => _currentNode;
         }
+
+        public double getTotalDistance()
```

Comment:
This is not used.

If it needs to be used in the future, it should be a getter instead